src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -216,7 +220,12 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
           },
         );
         setIsShown(false);
-        amplify.publish(WebAppEvents.CONVERSATION.SHOW, conversationEntity, {});
+
+        if ('key' in event) {
+          createNavigateKeyboard(generateConversationUrl(conversation.qualifiedId), true)(event);
+        } else {
+          createNavigate(generateConversationUrl(conversation.qualifiedId))(event);
+        }
```

Comment:
Conversation show logic stays the same.

`ContentViewModel` was listening for `WebAppEvents.CONVERSATION.SHOW` event and was calling `showConversation` with passed arguments (see https://github.com/wireapp/wire-webapp/blob/dev/src/script/view_model/ContentViewModel.ts#L118). This is exactly what happens after changing the route, see: https://github.com/wireapp/wire-webapp/blob/dev/src/script/page/AppMain.tsx#L167